CT_BarMod/CT_BarMod_Use.lua

```diff
@@ -824,11 +824,11 @@ end
 local function CT_BarMod__ActionButton_HideOverlayGlow(self)
 	-- This is a modified version of ActionButton_HideOverlayGlow from ActionButton.lua
 	if ( self.overlay ) then
-		if ( self.overlay.ProcStartAnim:IsPlaying() ) then
-			self.overlay.ProcStartAnim:Stop();
+		if ( self.overlay.animIn:IsPlaying() ) then
```

Comment:
I don't know what you want to do about this stuff. it definitely seems to work for cata, but probably breaks the other versions? I'm not really sure since classic doesn't have overlay glow and I don't play retail

Reply:
This appears like a partial reversion of 8b5ebc3e90021e56efd7e852e1d7e90954322368 that was contributed to make the glows work in retail.   It can definitely be made to work, if Blizzard has indeed chosen to keep the two out of sync.

Reply:
I'm not sure, but it was giving me nils when spells were proccing and the overlay glow wasn't working. The issue was reported by another user and I looked at the source code for ActionButton of the 4.4.0 tag and that's what I based my fix on

Reply:
Does this work?  b764406fdcf1da51340ca88285dbee37a84a1edd

(I can't easily generate a glow effect on cata classic.)

Reply:
I can try to test it later. I'm not at my computer right now

Reply:
it seems like it's working. no lua errors and I got the expected glow effect behavior